athenahealth/documents.go

```diff
@@ -181,6 +181,111 @@ func (h *HTTPClient) AddDocument(ctx context.Context, patientID string, opts *Ad
 	return res.DocumentID, nil
 }
 
+type AddClinicalDocumentOptions struct {
+	AttachmentContents []byte
+	AttachmentType     *string
+	AutoClose          *string
+	ClinicalProviderID *int
+	DepartmentID       *int
```

Comment:
DepartmentID appears to be a mandatory field (blue bar on the side) so should not be a pointer https://docs.athenahealth.com/api/api-ref/document-type-clinical-document#Add-clinical-document-to-patient's-chart

Reply:
👍